packages/@react-spectrum/icon/docs/workflow-icons.mdx

```diff
@@ -60,6 +60,21 @@ import Beaker from '@spectrum-icons/workflow/Beaker';
 <Beaker size="XXL" />
 ```
 
+## Coloring
+
+Icons support four Spectrum semantic colors: `negative`, `notice`, `positive`, and `informative`. Icons within React Spectrum components typically
+have their colors styled appropriately, but you can use the `color` prop to adjust the color of any standalone icons.
+
+```tsx example
+import Alert from '@spectrum-icons/workflow/Alert';
+
+<Alert />
```

Comment:
Should we wrap these in a Flex with a gap so they aren't quite that close together?
Also we should put an aria-label on them, otherwise we'll likely get some warnings